README.md

```diff
@@ -13,24 +15,64 @@ Install
 #### <i class="icon-file"></i> Composer
 Install package via composer:
 
-    $ composer require opstalent/crud
+```bash
+$ composer require opstalent/crud
+```
 
 Enable `OpstalentCrudBundle` in `/app/AppKernel.php`
 
-    $bundles = [
-	 ...
-	 new Opstalent\CrudBundle\OpstalentCrudBundle(),
-	 ...
-    ];
+```php
+$bundles = [
+    new Opstalent\CrudBundle\OpstalentCrudBundle(),
+];
+```
+
 
 ----------
 
 Documentation
 -------------
 
+####Entity Annotation
```

Comment:
IMO structure should be like this:

#### Annotations
##### `Entity`
##### `Field`

Reply:
fixed